framework/src/modules/chain/logic/in_transfer.js

```diff
@@ -82,8 +87,13 @@ InTransfer.prototype.calculateFee = function() {
  * @returns {SetImmediate} error
  * @todo Add description for the params
  */
-InTransfer.prototype.verify = function(transaction, sender, cb, tx) {
-	const lastBlock = modules.blocks.lastBlock.get();
+InTransfer.prototype.verify = async (transaction, sender, cb, tx) => {
```

Comment:
This function converted to `async`, we should declare all `verify` functions to async.